Celeste.Mod.mm/Patches/MountainModel.cs

```diff
@@ -112,7 +124,13 @@ class patch_MountainModel : MountainModel {
                         newMountain = meta.Mountain;
                     }
                 }
-                string oldPath = Path.Combine("Maps", PreviousSID ?? "").Replace('\\', '/');
+                string oldPath;
+                try {
+                    path = Path.Combine("Maps", PreviousSID ?? "").Replace('\\', '/');
+                }
+                catch (ArgumentException) {
+                    path = "Maps";
+                }
```

Comment:
This change seems to cause a build failure. Did you mean to assign to `oldPath` instead of `path` here?

Reply:
you ever make a typo in your first pr to a repo that more than 2 people work on